### PR TITLE
add profile task execute log

### PR DESCRIPTION
**Pull request by mrproliu**
add profile task instance execute log

### CODE REVIEW

File: profile.graphqls [hunk 1]
```diff
@@ -39,6 +39,22 @@ type ProfileTaskCreationResult {
     id: String
 }
 
+# Profile task execute log
+type ProfileTaskLog {
+    id: String!
+    # profile task id
+    taskId: String!
+    # execute instance id
+    instanceId: ID!
```
Comment (wu-sheng) [inline profile.graphqls]: Add name here too.

File: profile.graphqls [hunk 2]
```diff
@@ -39,6 +39,22 @@ type ProfileTaskCreationResult {
...
+    # operation type
+    # 1: when sniffer has notified
+    # 2: when sniffer has execute finish to report
+    operationType: Int!
```
Comment (wu-sheng) [inline profile.graphqls]: Are you expecting two log records, one for notification, the other for finished? As there is no update for this entity in the backend.

File: profile.graphqls [hunk 3]
```diff
@@ -54,6 +70,9 @@ type ProfileTask {
     minDurationThreshold: Int!
     # when start monitor, time interval for each dumping the stack
     dumpPeriod: Int!
+
+    # instance operation logs
+    logs: [ProfileTaskLog]
```
Comment (wu-sheng) [inline profile.graphqls]: `[ProfileTaskLog]` -> `[ProfileTaskLog!]!`

File: profile.graphqls [hunk 4]
```diff
@@ -39,6 +39,22 @@ type ProfileTaskCreationResult {
     id: String
 }
 
+# Profile task execute log
+type ProfileTaskLog {
+    id: String!
+    # profile task id
+    taskId: String!
```
Comment (wu-sheng) [inline profile.graphqls]: Task ID has been included in the `ProfileTask` entity. Please remove this.

File: profile.graphqls (hunk 2 above)
Comment (mrproliu) [reply profile.graphqls]: Yes, there will have two log records. No update operation on the database.

Comment (wu-sheng) [reply profile.graphqls]: operationType should be a type, rather than an int.

Comment (mrproliu) [reply profile.graphqls]: Changed that. Please have a look is that ok.

File: profile.graphqls [hunk 5]
```diff
@@ -39,6 +39,27 @@ type ProfileTaskCreationResult {
     id: String
 }
 
+# Profile task log operation type
+enum ProfileTaskLogOperationType {
+    # when sniffer has notified
+    NOTIFIED,
+    # when sniffer has execute finish to report
+    EXECUTE_FINISH
```
Comment (wu-sheng) [inline profile.graphqls]: `EXECUTE_FINISH` -> `EXECUTION_FINISHED`, also comments should be changed.

**Pull request merged**